src/main/resources/db/migration/V5.6.0.1__7440-configurable-license-list.sql

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
I don't see how to comment on the previous file - why the namechange from the unique-constraint-storageidentifier.sql script?

Reply:
@qqmyers that probably has to do with deploying different dataverse branches and versions and the flyway table getting all mixed up